### DISCUSSION TITLE
How to set instant in startat activity?

### DISCUSSION THREAD

**Discussion by Rummy99** [elsa-workflows/elsa-core#3362] [Q&A]
What input StartAt activity can accept?

I have tried few options. It's not working for me. Workflow instance goes to faulted state. Can share some examples?

Thank you.

**Comment by sfmskywalker** [comment]
The StartAt activity doesn't expect any input. It's just a trigger that causes the workflow to start or resume at some specified instant in the future.

What you can do of course is set a variable to whatever input you wanted to provide, and read this variable after the StartAt activity.

You mentioned you tried a few options. If you elaborate what you tried specifically I might have a better understanding of what you're looking to do and offer some suggestions.

**Reply by Rummy99** [reply]
Hi Sipke,

Sorry for the confusion. My query was regarding the Instant value that a startat activity can accept.
I have corrected the title now.

I need to set the start at timer to trigger exactly after 10 hours. I am using UI designer to design the workflow and JavaScript expression. I managed to achieve by setting the Instant like below. Please help to share if there are other ways to achieve it.

currentInstant().Plus(Duration. FromHours (10));

Thank you.
